_utils/run.sh

```diff
@@ -0,0 +1,2 @@
+python slave_refresh.py
```

Comment:
No, because the [host_refresh.py](https://github.com/mozilla-services/services-test/blob/bug/1223283/_utils/slave_refresh.py#L15) file generates the [.env](https://github.com/mozilla-services/services-test/blob/bug/1223283/_utils/firefox_env_handler.py#L113) file from the .ini config files.